cmd/cmdutil/helpers.go

```diff
@@ -47,6 +47,16 @@ func AddLimitFlag(cmd *cobra.Command, limit int, desc string) {
 	cmd.PersistentFlags().IntP("limit", "l", limit, desc)
 }
 
+// AddMaxConcurrencyFlag adds --max-concurrency to command
+func AddMaxConcurrencyFlag(cmd *cobra.Command, max int, desc string) {
+	cmd.PersistentFlags().IntP("max-concurrency", "m", max, desc)
+}
+
+// AddMaxErrorsFlag adds --max-errors to command
+func AddMaxErrorsFlag(cmd *cobra.Command, max int, desc string) {
+	cmd.PersistentFlags().IntP("max-errors", "e", max, desc)
```

Comment:
Do we want to have shorthand flags for these?

Reply:
I removed. But I'm fine with it either way